extension/surveys/survey-generator.js

```diff
@@ -571,3 +571,45 @@ function getFormValues(questionArr, form) {
   }
   return responses;
 }
+
+/**
+ * Checks whether a form has any responses or text in it.
+ * @param {object} form The form to check.
+ * @returns {boolean} True if the form has content.
+ */
+function formHasContent(form) {
```

Comment:
I'm trying to stupid-proof/future-proof this so I'm gonna stick with this one in case new elments get added later w/o the html class on it